ios/RNPhotosFramework/PHAssetsService.m

```diff
@@ -148,20 +159,53 @@ +(NSMutableDictionary *)extendAssetDictWithAssetResourcesMetadata:(NSMutableDict
         if(mimeTypeCString != nil) {
             mimeType = (__bridge NSString *)(mimeTypeCString);
         }
-        
-        [arrayWithResourcesMetadata addObject:@{
-                                                     @"originalFilename" : resourceMetadata.originalFilename,
-                                                     @"assetLocalIdentifier" : resourceMetadata.assetLocalIdentifier,
-                                                     @"uniformTypeIdentifier" : resourceMetadata.uniformTypeIdentifier,
-                                                     @"type" : [[RCTConvert PHAssetResourceTypeValuesReversed] objectForKey:@(resourceMetadata.type)],
-                                                     @"mimeType" : mimeType,
-                                                     @"fileExtension" : [resourceMetadata.originalFilename pathExtension]
-                                                     }];
+        [self videoUrlForLivePhotoAsset:asset andVideoResource:resourceMetadata withCompletionBlock:^(NSURL *url) {
+            [arrayWithResourcesMetadata addObject:@{
+                                                    @"originalFilename" : resourceMetadata.originalFilename,
+                                                    @"assetLocalIdentifier" : resourceMetadata.assetLocalIdentifier,
+                                                    @"uniformTypeIdentifier" : resourceMetadata.uniformTypeIdentifier,
+                                                    @"type" : [[RCTConvert PHAssetResourceTypeValuesReversed] objectForKey:@(resourceMetadata.type)],
+                                                    @"mimeType" : mimeType,
+                                                    @"fileExtension" : [resourceMetadata.originalFilename pathExtension],
+                                                    @"videoUrl": url ? [url absoluteString] : @""
```

Comment:
Does the empty string for `videoUrl` pass the bridge? If this is the case I think we can replace it with `NSNull()` which translates to undefined in JS.

Reply:
Thanks, I changed it to make use of `[NSNull null]` and is now returning a null value instead of an empty string